### ISSUE TITLE
ISOs also released via Bittorrent

### ISSUE THREAD

**Issue by darkain** [clonos/cp#16]
Would it be possible for current/future releases of the ISO image to be made available through Bittorent?

Right now I'm attempting to download the ISO, but it is only going at ~80KB/sec, estimated ~4 hours to download. 

I'm on gigabit internet and have a 24/7 torrent seeding jail setup that is dedicated to various Linux and FreeBSD distributions, and would fully support this distro as well. This distro is also listed on DistroWatch, so I'm sure others would as well: https://distrowatch.com/table.php?distribution=clonos

**Comment by olevole** [comment]
I agree, it is absolutely 'must have'. In addition, I would really like to use mirror services (like any other distros which are located, for example, at http://mirror.easyspeedy.com/ , http://mirror.truenetwork.ru , http://mirror.nl.leaseweb.net/ ...) , but I do not know if there are any free services. it requires some study

**Comment by y2kbadbug** [comment]
@darkain After discussing through email with olevole I am working on setting up two mirrors, one in Canada and one in Oregon, United States.

There is actually a torrent on distrowatch: https://distrowatch.com/dwres/torrents/clonos-19.01-release.iso.torrent

**Comment by olevole** [comment]
I set up rsync for those who can and are ready to provide a mirror in their region:

`rsync rsync://electro.bsdstore.ru/clonos/`

I can post your mirror on the site if you can provide a mirror. Thank you!

**Comment by darkain** [comment]
I've added the torrent to my seeding server. Once I have the files, my system will seed publicly on fast internet!
